apps/api/src/request/router.js

```diff
@@ -46,20 +47,38 @@ router.get('/:id', async ctx => {
 
 router.put("/:id", async ctx => {
     const user = ctx.state.user;
-    const updatedData = ctx.request.body;
+    let updatedData = ctx.request.body;
 
     const request = await getOne({
         client: ctx.state.db,
-        id: ctx.params.id,
+        id: ctx.params.id
     });
 
     if (!isAuthorized(user, request, updatedData)) {
-        const error = new Error("Forbidden.");
+        const error = new Error("Forbidden");
         error.status = 403;
 
         throw error;
     }
 
+    if (
+        request.status === 'MANAGEMENT_BUILDING' &&
+        updatedData.deliveryTracking
+    ) {
+        const quantitiesDelivered = extractQuantitiesFromDeliveries(
+            updatedData.deliveryTracking
+        );
+        updatedData.maskSmallSizeDeliveredQuantity = quantitiesDelivered.small;
+        updatedData.maskLargeSizeDeliveredQuantity = quantitiesDelivered.large;
+
+        if (
+            quantitiesDelivered.small >= request.maskSmallSizeQuantity &&
```

Comment:
Shouldn't we trigger and error when it's `>`?

Reply:
We discussed it a little with @tmaziere, and in fact we said that if they wanted to deliver more, it had to be possible!